client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/packages-step/style.scss

```diff
@@ -131,13 +131,9 @@
 	margin-bottom: 10px;
 	float: left;
 	margin-right: 15px;
-}
 
-@-moz-document url-prefix('') {
-	/* fix the width in Firefox */
-	.packages-step__package-weight {
-		width: 135px;
-		margin-right: 65px;
+	input {
+		min-width: 0;
```

Comment:
What's the reason behind styling `input` like this?

Reply:
As I understand it (somewhat based on [this StackOverflow answer](https://stackoverflow.com/a/36247448)), is that the input is being set to `100%` width [here](https://github.com/trey/wp-calypso/blob/3d61007729e10d24022f5665474dc8c2d821f4cb/assets/stylesheets/shared/_extends-forms.scss#L16) and is therefor stretching the entire `180px` width of `.packages-step__package-weight` in spite of the fact that it's inside of a flex container. Setting the `min-width` to `0` lets it shrink as needed.

As I mentioned on the ticket, I was tempted to add the fix [here](https://github.com/Automattic/wp-calypso/blob/fbf86b24af35cb6882a590063252d80d95dd62cf/client/components/forms/form-text-input-with-affixes/style.scss#L14-L18), but I'd be afraid it would have unintended side-effects.